<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>364</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <widget class="QFrame" name="frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="LoginButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Login</string>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/icons/login</normaloff>:/icons/login</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Link">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="usernameLabel">
          <property name="toolTip">
           <string>please input user name.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>username.</string>
          </property>
          <property name="text">
           <string>&amp;UserName:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>usernameLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="passwordLabel">
          <property name="toolTip">
           <string>please input password.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>password.</string>
          </property>
          <property name="text">
           <string>&amp;Password:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>usernameLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="title_label"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="titleLabel">
          <property name="font">
           <font>
            <family>Aharoni</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>user info</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring></cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="addressbookLabel">
          <property name="font">
           <font>
            <family>Aharoni</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>addresses book</string>
          </property>
          <property name="text">
           <string>address book</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring></cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_uid">
          <property name="toolTip">
           <string>UID.</string>
          </property>
          <property name="text">
           <string>UID:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>usernameLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="UIDLabel"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_Address1">
          <property name="toolTip">
           <string>Address.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="buddy">
           <cstring>addressLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>status:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="statusLabel">
          <property name="toolTip">
           <string>status.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_Address3">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="buddy">
           <cstring>addressLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_email">
          <property name="toolTip">
           <string>Email.</string>
          </property>
          <property name="text">
           <string>Email:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>emailLabel</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="emailLabel">
          <property name="toolTip">
           <string>email.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="blank_label1"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="blank_label2"/>
        </item>
        <item row="6" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="SubscriptButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Subscription</string>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/icons/wallet</normaloff>:/icons/wallet</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="6" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="LogoutButton">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&amp;Logout</string>
            </property>
            <property name="icon">
             <iconset resource="../bitcoin.qrc">
              <normaloff>:/icons/logoff</normaloff>:/icons/logoff</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="blank_label3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>usernameLabel</tabstop>
  <tabstop>passwordLabel</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>SubscriptButton</tabstop>
  <tabstop>LogoutButton</tabstop>
 </tabstops>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
